react/components/ProductSummaryBuyButton/ProductSummaryBuyButton.js

```diff
@@ -14,6 +16,18 @@ import displayButtonTypes, {
 
 import productSummary from '../../productSummary.css'
 
+function checkSKUAvailability(seller) {
+  return path(['commertialOffer', 'AvailableQuantity'], seller) > 0
+}
+
+function checkAvailableSKUsInSellers(item) {
+  return item.sellers && item.sellers.filter(checkSKUAvailability).length > 0
+}
+
+function hasOneSKU(items) {
```

Comment:
```suggestion
function hasOnlyOneSKU(items) {
```

The `Only` in the name makes it clearer that this not true when there are more than one SKU